<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjDlg</class>
 <widget class="QDialog" name="AddObjDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Grasping Object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="partInfoGroupBox">
     <property name="title">
      <string>Part Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="partName">
          <property name="text">
           <string>Part Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="partNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="fileName">
          <property name="text">
           <string>File Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameLineEdit">
          <property name="text">
           <string>C:\</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseBtn">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="PAgroupBox">
     <property name="title">
      <string>Position and Attitude</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="PosAttRBtn">
        <property name="text">
         <string>Using Position and Attitude</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="XLabel">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="XLineEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="YLabel">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="YLineEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ZLabel">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ZLineEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="Heading">
          <property name="text">
           <string>Heading</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HLineEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Pitch">
          <property name="text">
           <string>Pitch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PLineEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Roll">
          <property name="text">
           <string>Roll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RLineEdit">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Matrix</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="MatRBtn">
        <property name="text">
         <string>Using Matrix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input 4x4 elements divided by comma: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>a01,a02,a03,a04,a05,a06,a07,a08,a09,...,a16</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MatrixLineEdit">
        <property name="text">
         <string>0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
